pkg/raft/rafttest/node_test.go

```diff
@@ -137,7 +137,7 @@ func TestPause(t *testing.T) {
 
 func waitLeader(ns []*node) int {
 	var l map[pb.PeerID]struct{}
-	var lindex int
+	var lindex = -1
```

Comment:
Should this be reset to `-1` on each iteration? Might be better to move the definition inside the loop then.

Reply:
Done. I also moved to clearing the map on each iteration, instead of re-allocating it.